src/error.rs

```diff
@@ -45,6 +45,14 @@ pub enum Error {
     UrlEncode(#[source] serde_urlencoded::ser::Error),
 }
 
+// impl std::error::Error for Error {
+//     fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+//         match self {
+//             Error::InvalidCharacterCount(, )
+//         }
+//     }
+// }
+
```

Comment:
```suggestion
```

No need to keep this if it's not being used.

Reply:
Sorry, this was an oversight on my part